enterprise/releases/2024-10-0.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: '2024.10.0 Release'
+backlink_href: /docs/enterprise/releases
+backlink_title: 'pganalyze Enterprise Server: Release Changelog'
+---
+
+**Docker tag:** `quay.io/pganalyze/enterprise:v2024.10.0`
+
+## Notes
+
+- This release requires a database migration when coming from older releases, be sure to follow the [recommended upgrade steps](/docs/enterprise/upgrade)
+- **Important:** If you were previously running a version older than v2024.04.X we highly recommend first upgrading to v2024.04.5 and staying there for 7 to 35 days
+  - The v2024.04.0 release introduced a backend change for how query statistics are stored. Starting that release the old format was still supported in a dual write mode.
+  - Starting with this release the new packed format is required and the only one supported.
+  - If you have not been on v2024.04.X for sufficient time the migrations will fail with an error to prevent accidental loss of query statistics.
```

Comment:
Should we note the optional override here? If not, users who read the release notes may assume there is no direct upgrade path (even if you're willing to lose that data). I think that's okay, just calling it out.

Reply:
I think its okay to not mention it since I'd rather not have people set that by accident, and the warning is directly telling you what to do.